### PR TITLE
Cut v0.19.0-rc.0 🎉 

### PR DESCRIPTION
**Pull request by bwplotka**
Signed-off-by: Bartlomiej Plotka <bwplotka@gmail.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,27 +10,35 @@ NOTE: As semantic versioning states all 0.y.z releases can contain breaking chan
...
+## [v0.19.0-rc.0](https://github.com/thanos-io/thanos/releases/tag/v0.19.0-rc.0) - 2021.03.02
 
 ### Added
 
-- [#3740](https://github.com/thanos-io/thanos/pull/3740) Query: Added `--query.default-step` flag to set default step.
-- [#3700](https://github.com/thanos-io/thanos/pull/3700) ui: make old bucket viewer UI work with vanilla Prometheus blocks
+- [#3700](https://github.com/thanos-io/thanos/pull/3700) Compact/Web: Make old bucket viewer UI work with vanilla Prometheus blocks.
+- [#3657](https://github.com/thanos-io/thanos/pull/3657) *: It's now possible to configure HTTP transport options for S3 client.
+- [#3752](https://github.com/thanos-io/thanos/pull/3752) Compact/Store: Added `--block-meta-fetch-concurrency` allowing to configure number of go routines for block metadata synchronization.
 - [#2641](https://github.com/thanos-io/thanos/issues/2641) Query Frontend: Added `--query-range.request-downsampled` flag enabling additional queries for downsampled data in case of empty or incomplete response to range request.
```
Comment (yeya24) [inline CHANGELOG.md:20]: What about setting this to the correct PR https://github.com/thanos-io/thanos/pull/3723 instead of the issue?

Comment (bwplotka) [reply CHANGELOG.md:20]: Does it make a bit difference? I can if we want (:

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,27 +10,35 @@ NOTE: As semantic versioning states all 0.y.z releases can contain breaking chan
...
-- [#3031](https://github.com/thanos-io/thanos/pull/3031) Compact/Sidecar/other writers: added `--hash-func`. If some function has been specified, writers calculate hashes using that function of each file in a block before uploading them. If those hashes exist in the `meta.json` file then Compact does not download the files if they already exist on disk and with the same hash. This also means that the data directory passed to Thanos Compact is only *cleared once at boot* or *if everything succeeds*. So, if you, for example, use persistent volumes on k8s and your Thanos Compact crashes or fails to make an iteration properly then the last downloaded files are not wiped from the disk. The directories that were created the last time are only wiped again after a successful iteration or if the previously picked up blocks have disappeared.
-- [#3686](https://github.com/thanos-io/thanos/pull/3686) Query: Added federated metric metadata support.
+???? -[#3579](https://github.com/thanos-io/thanos/pull/3579) Cache: Added inmemory cache for caching bucket.
```
Comment (squat) [inline CHANGELOG.md:21]: What's ???? ?

File: CHANGELOG.md [hunk 3]
```diff
@@ -10,27 +10,35 @@ NOTE: As semantic versioning states all 0.y.z releases can contain breaking chan
...
+- [#3792](https://github.com/thanos-io/thanos/pull/3792) Receiver: Added `--tsdb.allow-overlapping-blocks` flag to allow overlapping tsdb blocks and enable vertical compaction.
+- [#3740](https://github.com/thanos-io/thanos/pull/3740) Query: Added `--query.default-step` flag to set default step. Useful when your tenant scrape interval is stable and far from default UI's 1s.
+- [#3686](https://github.com/thanos-io/thanos/pull/3686) Query,Sidecar: Added metric metadata API support. You can now configure you Querier to fetch Prometheus metrics metadata from leaf Prometheus-es!
```
Comment (squat) [inline CHANGELOG.md:24]: Nit: for consistency ```suggestion
- [#3686](https://github.com/thanos-io/thanos/pull/3686) Query/Sidecar: Added metric metadata API support. You can now configure you Querier to fetch Prometheus metrics metadata from leaf Prometheus-es!
```

File: CHANGELOG.md (hunk 1 above)
Comment (bwplotka) [reply CHANGELOG.md:20]: We can fix later 🤗

**Review by squat — APPROVED** [review]
Two small nits, otherwise lgtm

File: CHANGELOG.md (hunk 2 above)
Comment (bwplotka) [reply CHANGELOG.md:21]: uuuups

**Comment by bwplotka** [comment]
Will add in next PR, thanks @squat

File: CHANGELOG.md (hunk 2 above)
Comment (bwplotka) [reply CHANGELOG.md:21]: I wanted to double check this as it was removed from changelog somehow

**Comment by bwplotka** [comment]
Addressed all https://github.com/thanos-io/thanos/pull/3865

**Pull request merged**